jwst/tests_nightly/general/nirspec/test_nrs_ifu_wcs.py

```diff
@@ -0,0 +1,51 @@
+import os
+import pytest
+from numpy.testing import assert_allclose
+from gwcs.wcstools import grid_from_bounding_box
+
+from jwst.assign_wcs import AssignWcsStep, nirspec
+
+from jwst.datamodels import ImageModel
+
+pytestmark = [
+    pytest.mark.usefixtures('_jail'),
+    pytest.mark.skipif(not pytest.config.getoption('bigdata'),
+                       reason='requires --bigdata')
+]
+
+testdata = [
+    ('nrs1', 'jw00011001001_01120_00001_NRS1_rate.fits', 'jw00011001001_01120_00001_NRS1_assign_wcs.fits'),
+    ('nrs1_opaque', 'jw00011001001_01120_00001_NRS1_rate_opaque.fits', 'jw00011001001_01120_00001_NRS1_rate_opaque_assign_wcs.fits'),
+    ('nrs2', 'NRSIFU-COMBO-030_NRS2_SloperPipeline.fits', 'NRSIFU-COMBO-030_NRS2_SloperPipeline_assign_wcs.fits')
+]
+
+@pytest.mark.parametrize("test_id, input_file, truth_file", testdata)
+def test_nirspec_ifu_wcs(_bigdata, test_id, input_file, truth_file):
+    """
+    Regression test of creating a WCS object and doing pixel to sky transformation.
+    """
+    del test_id
+
+    input_file = os.path.join(_bigdata, 'nirspec', 'test_wcs', 'nrs1-ifu', input_file)
+    truth_file = os.path.join(_bigdata, 'nirspec', 'test_wcs', 'nrs1-ifu', truth_file)
+
+    result = AssignWcsStep.call(input_file,
+                       save_results=True,
+                       suffix='assign_wcs'
+                       )
+    result.close()
+
+    im = ImageModel(result.meta.filename)
+    imref = ImageModel(truth_file)
```

Comment:
Strictly speaking ifu files should be opened with `IFUImageModel` but I think `ImageModel` will work in this case as well. 
Do you mind adding a for loop and checking a few more slices, for example [0, 9, 16, 23, 29].  This test runs only on slice 0. I guess it hasn't been updated for a very long time.

Reply:
Good to know.  If you'd like to make that change in your PR adding more slices, I'll leave that to you.